### DISCUSSION TITLE
Feedback for experimental HTTP load balancing support

### DISCUSSION THREAD

**Discussion by SchoNie** [nginx-proxy/nginx-proxy#2195] [Ideas]
Pull request #2173 added experimental support for load balancing between the proxy and upstream server groups. If you use this feature, or if you would like changes to the feature before you start using it, please provide feedback here so we know whether it is working properly and can be promoted to officially supported.

**Comment by aheider** [comment]
Hi,
I definetely need this feature, especially the basic auth via htpasswd files!

Thanks so much.

**Reply by SchoNie** [reply]
Both are implemented.
https://github.com/nginx-proxy/nginx-proxy#basic-authentication-support

https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#upstream-server-http-load-balancing-support

**Comment by aristofun** [comment]
This is needed for sure!

**Reply by SchoNie** [reply]
This is implemented since a year: https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#upstream-server-http-load-balancing-support
